vendor/src/github.com/docker/distribution/digest/verifiers.go

```diff
@@ -1,8 +1,6 @@
 package digest
```

Comment:
@NathanMcCauley Could you submit a PR to distribution to make this fix? Are timing attacks problematic on short strings in this kind of context? If required, we may need to create a version for strings that won't require a copy of the data as a slice to perform the comparison.

This may be outside the scope of this PR.